models/models.py

```diff
@@ -0,0 +1,26 @@
+from odoo import api, models, fields, _
+import datetime as dt
+from odoo.exceptions import ValidationError
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+    active = fields.Boolean(default=True)
+    lifespan = fields.Integer(string='Lifespan')

```

Comment:
Add a new setting to **_Purchase > Configurations > Settings_** instead of adding it for each Purchase Order because we'll have one` lifespan` for all of our purchase orders